### PR TITLE
Update tokens and remove colors.textPlaceholder

### PR DESCRIPTION
**Pull request by jmartinesp**
What the title says.

### CODE REVIEW

File: compound/src/main/kotlin/io/element/android/compound/tokens/generated/internal/DarkColorTokens.kt [hunk 1]
```diff
@@ -30,315 +30,315 @@ import io.element.android.compound.annotations.CoreColorToken
...
+    val colorAlphaPink500 = Color(0xff6b0036)
+    val colorAlphaPink600 = Color(0x75fb0473)
+    val colorAlphaPink700 = Color(0x94fd1277)
+    val colorAlphaPink800 = Color(0xccfe1b79)
+    val colorAlphaPink900 = Color(0xf5fe4382)
+    val colorAlphaPink1000 = Color(0xfaff6691)
+    val colorAlphaPink1100 = Color(0xfffe86a4)
+    val colorAlphaPink1200 = Color(0xffffadc0)
+    val colorAlphaPink1300 = Color(0xffffd1db)
+    val colorAlphaPink1400 = Color(0xffffebef)
```
Comment (bmarty) [inline compound/src/main/kotlin/io/element/android/compound/tokens/generated/internal/DarkColorTokens.kt:341]: A shame that this cannot be diff easily. We should probably sort the `val` alphabetically.

**Review by bmarty — APPROVED** [review]
LGTM, thanks!

File: compound/src/main/kotlin/io/element/android/compound/tokens/generated/internal/DarkColorTokens.kt (hunk 1 above)
Comment (jmartinesp) [reply compound/src/main/kotlin/io/element/android/compound/tokens/generated/internal/DarkColorTokens.kt:341]: Maybe there's something we can add in https://github.com/element-hq/compound-design-tokens for that, but I somehow doubt it'll be that easy.

**Comment by jmartinesp** [comment]
Actually, it wasn't that hard: https://github.com/element-hq/compound-design-tokens/pull/125

**Pull request merged**